client/src/utils/fetchInterviews.js

```diff
@@ -0,0 +1,34 @@
+import { getUserInterviews, getQuestion, getUser } from './apiFunctions'
+
+export const fetchInterviews = async (userId) => {
+  const { data: userInterviews } = await getUserInterviews(userId);
+  const interviews = [];
+  for (const interview of userInterviews) {
+    for (const user of interview.users) {
+      if (user.user === userId) {
+        const {
+          _id: userId,
+          firstName,
+          lastName
+        } = (await getUser(user)).data.user;
```

Comment:
I still don't think we need to be fetching user data here. We should already have it from our context, and this is a lot of API calls to do for every interview

Reply:
I went ahead and deleted all the logic of passing down user data (since we'll have access to it from state).